lang/en/local_csp.php

```diff
@@ -69,3 +69,14 @@
 $string['timeupdated'] = 'Last';
 $string['violateddirective'] = 'Policy';
 $string['privacy:metadata'] = 'The CSP plugin contains no user specific data.';
+
+# Notification modal
+$string['modalheader'] = 'External content detected';
+$string['modalbodystart'] = 'This page contains embedded external content.';
```

Comment:
Strictly this isnt correct, it could be internal but blocked for csp reasons. Can you just say 'blocked content' ?

Reply:
Sure, strictly speaking you are correct. I suppose in the majority of use cases it would be in fact external content, but the text should refer to the more general case.

However, using the word "blocked" in the header/first sentence would also be wrong because the popup can notify about sources not matching the `Content-Security-Policy-Report-Only` directives, in which case the content is **not** blocked.

I suppose the driving motivation behind the concept of CSP is for website admins to set their own rules as to what content sources/types they declare (potentially) **insecure**. (Regardless of whether or not a user of the site would agree with that classification.) Therefore I propose the following change to my original texts:

```php
$string['modalheader'] = 'Potentially insecure content detected';
$string['modalbodystart'] = 'This page contains embedded content that the administrators consider potentially insecure.';
```

This is followed up nicely by the distinction between sources that were actually blocked and those that were not (but reported to the admins) in the `modalbodyblocked` and `modalbodynotblocked` texts. Is that a good compromise?

Reply:
+1 sounds great